index.txt

```diff
@@ -913,7 +913,7 @@ as [Hahn2015]_, eq. (15)
     \breve{S}_n^m(\phi_\text{s},r_\text{s},\w) =
         -\pi \i^{m-n+1}
         \Hankel{2}{m}{\wc r_\text{s}}
-        Y_n^{-m}(\frac{\pi}{2},\phi_\text{s}).
+        Y_n^{-m}(0,\phi_\text{s}).
```

Comment:
\theta = 0 would be inside the horizontal plane, is this the intended direction?